pxtlib/blocks.ts

```diff
@@ -806,5 +808,20 @@ namespace pxt.blocks {
                 )
             })
         }
+
+        if (pxt.blocks.showBlockIdInTooltip) {
+            for (const id of Object.keys(_blockDefinitions)) {
+                const tooltip = _blockDefinitions[id].tooltip;
+                if (typeof tooltip === "object" && tooltip !== null) {
```

Comment:
Wondering if these conditions should be reversed for short circuiting or if the second check is needed. If the tooltip has type object, is it possible for it to be null?

Reply:
Weirdly, I believe typeof null will return object, so `typeof tooltip === "object"` would be true. I think it'd be fine to check for null first, but I don't think it'll make much difference.